src/paint/paint.c

```diff
@@ -59,17 +59,153 @@ bool sub_98196C(
 	uint16 z_offset,
 	uint32 rotation
 ) {
-	int flags = RCT2_CALLPROC_X(RCT2_ADDRESS(0x0098196C, uint32)[rotation],
```

Comment:
This function appears to be for sprites where bound box does not need offsets. Perhaps if we call 7c `push_sprite` this would be called something like `push_sprite_tight`.